src/redux/reducers/vendorReducer.js

```diff
@@ -0,0 +1,31 @@
+import {REGISTER_BUSINESS_FAIL,REGISTER_BUSINESS_SUCCESS} from '../actions/type';
+
+const initialState = {
+  token: localStorage.getItem('token'),
+  isAuthenticated: false,
+  isLoading: false,
+  user: null,
+  openloginModal: false,
+  openregisterModal: false
+}
+
+export default function(state= initialState,action){
+  switch(action.type){
+  case REGISTER_BUSINESS_FAIL:
+    localStorage.removeItem('token');
+    localStorage.removeItem('user');
```

Comment:
Don't log the user out on register business fail just set the msg and status which is received from the server.
If a status of 200 is not received the display a warning alert on the list your business page else a success alert.

Reply:
done